src/components/chart-scroll/ChartScroll.css

```diff
@@ -239,7 +239,30 @@ justify-content: center;
 }
 
 .csv {
-  margin-left: 1.5em;
+  
+  text-align: center;
+  margin-bottom: 2vh;
 }
 
-
+.modal {
+  background-color: white;
+  opacity: 95%;
+  height: 40vh;
+  margin: 20vh 20vw 0 20vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+.modal:focus {
+  outline: black 2px solid;
+}
+.modal button{
+  font-family: roboto-mono, Helvetica, Arial, sans-serif;
+  width: 50px;
```

Comment:
Could we reduce the width a bit?

Reply:
deleted the width: 50px and now it is just auto, which is probably fine.
![Screen Shot 2020-07-19 at 1 55 18 PM](https://user-images.githubusercontent.com/50382381/87885045-a6109180-c9c7-11ea-8e6f-54c78a8f552c.png)